db/crud.go

```diff
@@ -1375,7 +1375,7 @@ func (db *DatabaseCollectionWithUser) PutExistingCurrentVersion(ctx context.Cont
 				if len(opts.RevTreeHistory) > 0 && !opts.AlignRevTrees {
 					parent, currentRevIndex, err := db.revTreeConflictCheck(ctx, opts.RevTreeHistory, doc, opts.NewDoc.Deleted)
 					if err != nil {
-						base.InfofCtx(ctx, base.KeyCRUD, "conflict detected between the two HLV's for doc %s, incoming version %s, local version %s, and conflict found in rev tree history", base.UD(doc.ID), opts.NewDocHLV.GetCurrentVersionString(), doc.HLV.GetCurrentVersionString())
+						base.InfofCtx(ctx, base.KeyCRUD, "conflict detected between the two HLV's for doc %s, incoming version %#v, local version %#v, and conflict found in rev tree history", base.UD(doc.ID), opts.NewDocHLV.ExtractCurrentVersionFromHLV(), doc.HLV.ExtractCurrentVersionFromHLV())
```

Comment:
I think this logging should be dropped to at least Debug, and maybe moved to VV for test only logging.

What do we log for revtrees in this circumstance? It might just be that this is a 409 conflict as a rev message that 

```suggestion
						base.DebugfCTx(ctx, base.KeyCRUD, "conflict detected between the two HLVs for doc %s, incoming version %#v, local version %#v, and conflict found in rev tree history", base.UD(doc.ID), opts.NewDocHLV.ExtractCurrentVersionFromHLV(), doc.HLV.ExtractCurrentVersionFromHLV())
```

If we do log this I think we probably want not just CV, because without  full history we won't know why this conflicts.

Reply:
dropped to debug and removed hlv info.